homeassistant/components/onedrive/strings.json

```diff
@@ -29,6 +29,16 @@
       "default": "[%key:common::config_flow::create_entry::authenticated%]"
     }
   },
+  "issues": {
+    "drive_full": {
+      "title": "OneDrive data cap exceeded",
+      "description": "Your OneDrive has exceeded your quota limit. This means your next backup will fail. Please free up some space or upgrade your OneDrive plan. Currently using {used} GB of {total} GB."
+    },
+    "drive_almost_full": {
+      "title": "OneDrive near data cap",
+      "description": "Your OneDrive is near your quota limit. If you  go over this limit your drive will be temporarily frozen and your backups will start failing. Please free up some space or upgrade your OneDrive plan. Currently using {used} GB of {total} GB."
```

Comment:
How definite is "will start to fail" vs "are likely to fail"?
Just thinking about someone with a 500kb backup vs a 50mb backup, one has oodles of room with 60mb left, the other has one backup.

Reply:
Ah missed that, just saw the "upgrade or..."